DHL/JSON/66da0fe7-7a0b-4e36-b685-8f4d6b06607f.json

```diff
@@ -0,0 +1,531 @@
+                                      "Val": "Yes",
+                                      "DisplayTxt": "Yes",
+                                      "SrtOrd": 1,
+                                      "Nm": "Opt1"
+                                    },
+                                    {
+                                      "Val": "No",
+                                      "DisplayTxt": "No",
+                                      "SrtOrd": 2,
+                                      "Nm": "Opt2"
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "f5596d97-17a0-4f52-acbc-c0f598e57c3d",
+                                  "Nm": "Purpose Name - Radio button",
+                                  "Dsc": "Purpose Name with a radiobutton component",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Agree to Shipment cost more than 999 Euros",
+                                  "isExcl": false,
+                                  "isMan": true,
+                                  "Rul": [
+                                    {
+                                      "RulId": "e74fa4ae-a5f6-4d08-b583-f96373e3b958",
+                                      "Nm": "Not agree purpose",
+                                      "Cond": "**.Att[Id='6a2f893d-8c39-42ed-9b9e-450d21092907'].Val='Opt2'",
+                                      "AttrMtdt": [
+                                        {
+                                          "DflVal": "Not Agree to Shipment cost more than 999 Euros"
+                                        }
+                                      ]
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "6af68f3b-0c02-4c3a-8774-6132aa38d3d7",
+                                  "Nm": "Purpose Description - Radio button",
+                                  "Dsc": "Purpose Description with a radiobutton component",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Not agree to Shipment cost more than 999 Euros",
+                                  "isExcl": false,
+                                  "isMan": true,
+                                  "Rul": [
+                                    {
+                                      "RulId": "ed7d5c64-cab3-4b19-a4c7-13825ac3ee3d",
+                                      "Nm": "Not agree Purpose Description",
+                                      "Cond": "**.'6a2f893d-8c39-42ed-9b9e-450d21092907'].Val='Opt1'",
+                                      "AttrMtdt": [
+                                        {
+                                          "DflVal": "Agree to Shipment cost more than 999 Euros"
```

Comment:
value -> User confirmed that the shipment does not costs more than 999 Euros

Reply:
Done